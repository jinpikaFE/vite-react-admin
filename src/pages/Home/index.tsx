import ProCard from '@ant-design/pro-card';
import React from 'react';

const Home: React.FC = () => {
  return (
    <>
      <ProCard>Home</ProCard>
    </>
  );
};

export default Home;
