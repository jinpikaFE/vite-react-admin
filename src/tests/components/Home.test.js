import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Home from '../../pages/home';

afterEach(cleanup);

it('should take a snapshot', () => {
  const { asFragment } = render(<Home />);

  expect(asFragment()).toMatchSnapshot();
});
