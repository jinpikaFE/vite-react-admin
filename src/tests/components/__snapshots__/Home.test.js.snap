// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div>
    homessd
  </div>
</DocumentFragment>
`;
